DEVELOPMENT.md

```diff
@@ -42,53 +42,9 @@ Install these tools:
     export KO_DOCKER_REPO="kind.local"
     ```
 
-1. Install knative-eventing core and cert-manager, wait for them to be ready, and enable new-apiserversource-filters:
-    ```bash
-    export CERT_MANAGER_VERSION=v1.9.1
-    export KNATIVE_EVENTING_VERSION=v1.15.0
-
-    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
-    kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
-    kubectl rollout status deployment --namespace=cert-manager --timeout=30s
-    kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
-
-    kubectl apply -f - << EOF
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      name: config-features
-      namespace: knative-eventing
-      labels:
-        eventing.knative.dev/release: devel
-        knative.dev/config-propagation: original
-        knative.dev/config-category: eventing
-    data:
-      new-apiserversource-filters: enabled
-    EOF
-    ```
-
-1. Generate operator code:
-    ```bash
-    cmd/operator/generate.sh
-    ```
-
-1. Install a database:
-   ```bash
-   integrations/database/postgresql/install.sh
-   ```
-
-## Install KubeArchive
-
-1. Use Helm to install KubeArchive:
-   ```bash
-   helm install kubearchive charts/kubearchive --create-namespace -n kubearchive \
-       --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api) \
-       --set-string sink.image=$(ko build github.com/kubearchive/kubearchive/cmd/sink) \
-       --set-string operator.image=$(ko build github.com/kubearchive/kubearchive/cmd/operator)
-   ```
-1. Check that the KubeArchive deployments are Ready:
+## Install KubeArchive, its dependencies, and initialize the database
    ```bash
-   kubectl get -n kubearchive deployments
+   hack/quick-install.sh
```

Comment:
4 things related to this change:

1. Can you please add in line 14 that the `kubectl` version needed is >1.31 ?
2. If we are not providing the `helm install` command to deploy kubearchive, `helm upgrade` makes no sense anymore as we don't have a helm chart deployed anymore (just a plain yaml). Should we than provide different instructions? Maybe a `kind delete cluster` and then reapply the quick-install?
3. Same applies for the section "Uninstall KubeArchive"
4. Now the instructions for testing the API Server with curl and the instructions to use the CLI don't work anymore (because we aren't creating the test namespace and resources, i.e, we don't have a service account with permissions to query a k8s resource). To solve this I can create a separate test script.

Reply:
@maruiz93 , I have addressed 1, 2, and 3.

How do you want to handle 4?

Reply:
I've created https://github.com/kubearchive/kubearchive/pull/456

As it's shorter we can merge it before this one. Other possibility is that you include those changes in this PR.

Reply:
@maruiz93 , I merged https://github.com/kubearchive/kubearchive/pull/456.  Can this conversation be resolved?